src/app.ts

```diff
@@ -152,10 +152,11 @@ class App {
 		app.post('/did/update', IssuerController.updateValidator, new IssuerController().updateDid);
 		app.post('/did/deactivate/:did', IssuerController.deactivateValidator, new IssuerController().deactivateDid);
 		app.get('/did/list', new IssuerController().getDids);
-		app.get('/did/:did', new IssuerController().getDid);
+		app.get('/did/search/:did', new IssuerController().resolveDidUrl);
 
 		// Resource API
 		app.post('/resource/create/:did', IssuerController.resourceValidator, new IssuerController().createResource);
+   	 	app.get('/resource/search/:did/:resourceId', new IssuerController().getResource);
```

Comment:
This is a cascading failure throughout on the linter, but appending `resourceId` in the path is a bad idea. Maybe the context got lost between what I provided to @anikitinDSR vs what I meant. This should _just_ be `:did`. If the `resourceId` is known, there's zero point in putting any other query parameters.

In actual practice, `/did/search` and `/resource/search` are almost identical: they construct and send a request to the resolver. I suggested splitting up the parameters across the two different APIs since a lot of the DID-only parameters (e.g., `service`) are only valid for DIDs, without resources and vice versa. So instead of having a long list of parameters all under the `/did/search` path, I suggested splitting them so that the resource-specific ones were under `/resource/search` instead. For example, a query like `did:cheqd:mainnet:xxxxx?service=LinkedDomains&resourceName=Logo` is absolutely meaningless, and so to prevent confusion, wanted to split those up.

Reply:
Done ✅